src/test/java/com/commercetools/sync/commons/utils/SyncUtilsTest.java

```diff
@@ -26,6 +25,8 @@
 
 class SyncUtilsTest {
 
+  Map<String, String> idToKeyValueMap = new HashMap<>();
```

Comment:
The cache isn't clean up after running a chain of unit tests.
It creates dependencies between test cases. Can you try to clean up the cache for each unit test cases and see the influence?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/701/commits/6fde9c6afded1a6a8fae4d46d300090d04bc8111